wurst/objects/items/PlagueBomb.wurst

```diff
@@ -0,0 +1,163 @@
+package PlagueBomb
+
+// Standard library imports:
+import Assets
+import ItemObjEditing
+import ClosureEvents
+import ClosureTimers
+import ClosureForGroups
+import ClosuresForItems
+import ChannelAbilityPreset
+import StackNSplit
+
+// Local imports:
+import LocalObjectIDs
+import LocalAssets
+import UnitExtensions
+
+@configurable let PLAGUE_RANGE = 500
+
+constant PLAGUE_ABIL = compiletime(ABIL_ID_GEN.next())
+constant ABILITY_RADIUS = 350.
+constant ITEM_ABILITIES = commaList(PLAGUE_ABIL)
+
+constant TOOLTIP = "Plague Bomb"
+constant TOOLTIP_EXTENDED = "Rains down plague on target location. All meat and buildings in the area will become infected."
+constant DESCRIPTION = "Rains plague down on an area infecting all meat and buildings."
+
+constant MODEL = Buildings.barrelsUnit0
+constant ICON  = LocalIcons.btNPlagueBomb
+constant EFFECT = Abilities.plagueCloudCaster
+
+constant MISSILE_FLY_HEIGHT = 140.
+constant PROJECTILE_INTERVAL = 1/30
+constant EFFECT_DURATION = 8.
+
+class PlagueBomb extends ChannelAbilityPreset
+    construct(int newAbilityId)
+        super(newAbilityId, 1, true)
+        this.presetManaCost(lvl -> 0)
+        this.presetCooldown(lvl -> 0)
+        this.setAreaofEffect(1, ABILITY_RADIUS)
+        this.setHeroAbility(false)
+        this.setItemAbility(false)
+        this.presetTargetTypes(Targettype.POINT)
+        this.setName(TOOLTIP)
+        this.setBaseOrderID(1, "antimagicshell")
```

Comment:
ChannelAbilityPreset use a random orderID so it shouldn't be neccessary to set it unless you want to avoid conflicting orderID but I don't know if this can occur with item cast.

Reply:
Items use their own orders. Also don't manually set it like this because you break parts of `ChannelAbilityPreset` implementation. Update the banned orders (pending some problems there).